### PR TITLE
refactor: eslint setup

### PR DESCRIPTION
**Pull request by erisu**
### Motivation and Context

Code syntax cleanup

Closes #73

### Description

* replace dependencies with @cordova/eslint-config
* update eslint config
* eslint corrections

### Testing

- `npm t`
- `npm run eslint`

### Checklist

- [x] I've run the tests to see all new and existing tests pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,13 +38,13 @@
...
-    "eslint": "^5.0.0",
-    "eslint-config-semistandard": "^14.0.0",
-    "eslint-config-standard": "^13.0.1",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-node": "^8.0.0",
-    "eslint-plugin-promise": "^4.0.0",
-    "eslint-plugin-standard": "^4.0.0",
+    "eslint": "^6.5.1",
+    "eslint-config-semistandard": "^15.0.0",
+    "eslint-config-standard": "^14.1.0",
```
Comment (raphinesse) [inline package.json]: That needs to be `^13.0.1` unless `eslint-config-semistandard` publishes an update before we merge this.

File: src/PluginInfo/PluginInfo.js [hunk 2]
```diff
@@ -130,12 +131,13 @@ function PluginInfo (dirname) {
 
     function _parseConfigFile (tag) {
         var configFile =
-            { target: tag.attrib['target'],
-                parent: tag.attrib['parent'],
-                after: tag.attrib['after'],
+            {
```
Comment (raphinesse) [inline src/PluginInfo/PluginInfo.js]: Can we bring this up to the assignment line and unindent the object body please?

File: src/PluginInfo/PluginInfo.js [hunk 3]
```diff
@@ -149,9 +151,10 @@ function PluginInfo (dirname) {
 
     function _parseEditConfigs (tag) {
         var editConfig =
-            { file: tag.attrib['file'],
-                target: tag.attrib['target'],
-                mode: tag.attrib['mode'],
+            {
```
Comment (raphinesse) [inline src/PluginInfo/PluginInfo.js]: Can we bring this up to the assignment line and unindent the object body please?

File: package.json (hunk 1 above)
Comment (erisu) [reply package.json]: 👍 These dependencies will be drop soon and replaced with out new package anyways. It was just a step to get the PR in place and files linted.

**Comment by raphinesse** [comment]
Should we wait for `@cordova/eslint-config@2` before merging this?

**Comment by erisu** [comment]
Yeah, 

I think that will be best. 

I actually have been waiting for the `@cordova/eslint-config@2` to drop all the other dependencies. I will start preparing the `@cordova/eslint-config@2` vote today and hopefully we can get it released in 2 days.

File: .eslintrc.yml [hunk 4]
```diff
@@ -1,10 +1,2 @@
 root: true
-extends: semistandard
-rules:
-  indent:
-    - error
-    - 4
-  camelcase: off
-  no-unused-vars:
-    - error
-    - args: after-used
```
Comment (raphinesse) [inline .eslintrc.yml]: If repos choose to be stricter than our base, don't we want to maintain this for now? Maybe discuss bringing it into `@cordova/eslint-config` sometime?

Comment (erisu) [reply .eslintrc.yml]: I went ahead and re-added it but also updated the settings to be more accurate to what we want with `standards` defaults.

This change should still be discussed and brought to the `cordova-eslint` so all repos are enforced.

**Pull request merged**